src-docs/src/views/header/header.js

```diff
@@ -38,7 +38,7 @@ export default () => {
   const renderLogo = () => (
     <OuiHeaderLogo
       iconType="logoOpenSearch"
-      href="#"
+      href="https://oui.opensearch.org/latest/"
```

Comment:
Hover over the logo

<img width="600" alt="Screenshot 2023-04-18 at 09 07 36" src="https://user-images.githubusercontent.com/17855243/232660809-17287f87-fc5e-4f07-b1a8-66b47e75d596.png">